### DISCUSSION TITLE
Scale of ChargeHeatMap is wrong

### DISCUSSION THREAD

**Discussion by metawatchkalle** [teslamate-org/teslamate#2460] [General]
The Y-axis on the ChargeHeatMap is wrong. In my opinion it should limited to 100% because it is not possible to charge over 100%.
Setting the AxisScale / Y-Max to "100" solves the problem. (originally it is set to "auto").

**Comment by DrMichael** [comment]
Not sure, what you mean. The map is a map, what would be the Y-axis?!?

**Reply by metawatchkalle** [reply]
![Heatmap](https://user-images.githubusercontent.com/4587701/153402043-1b390734-0aba-457f-9783-cc7a6e433fe6.jpg)
See the arrow. 120% should be 100%

**Comment by DrMichael** [comment]
PR https://github.com/adriankumpf/teslamate/pull/2461
